sdk/js/src/EventEmitter.ts

```diff
@@ -39,7 +40,7 @@ export class EventEmitter {
     unsubscribe(key: string, handler?: eventHandler): void {
         checkParamType('key', key, 'string')
 
-        const eventNames = Object.keys(EventNames).map((e) => e.toLowerCase())
+        const eventNames = Object.values(EventNames)
         if (!eventNames.includes(key)) {
```

Comment:
kinda weird that the check here is different than the one on subscribe

Reply:
doesnt this mean you cant unsub from "variableUpdated:*"?

Reply:
good catch, i added made the check the same in both